testing/unit-tests.php

```diff
@@ -32,14 +23,14 @@
 require 'unit-tests/_files.php';
 
 // Setup test files
-$test = new GroupTest('CSSTidy unit tests');
+$test = new TestSuite('CSSTidy unit tests');
 foreach ($test_files as $test_file) {
 	require_once "unit-tests/$test_file";
 	list($x, $class_suffix) = explode('.', $test_file);
-	$test->addTestClass("csstidy_test_$class_suffix");
+	$test->add("csstidy_test_$class_suffix");
 }
 
 if (SimpleReporter::inCli()) $reporter = new TextReporter();
 else $reporter = new csstidy_reporter('UTF-8');
 
-$test->run($reporter);
+exit ($test->run($reporter) ? 0 : 1);
```

Comment:
Making sure to return a non-zero code when tests fail so that travis fails the build.